debugProtocol.json

```diff
@@ -457,6 +457,10 @@
 										"Debugger attached to an existing process.",
 										"A project launcher component has launched a new process in a suspended state and then asked the debugger to attach."
 									]
+								},
+								"pointerLength": {
+									"type": "number",
```

Comment:
Should we use `integer` instead of `number` for all the new fields that shouldn't be a floating point value?